devices/src/virtio/vhost/mod.rs

```diff
@@ -8,6 +8,7 @@
 //! Implements vhost-based virtio devices.
 
 use std;
+use std::io;
 use sys_util::Error as SysError;
```

Comment:
I see that 
```
/// Creating kill eventfd failed.
    CreateKillEventFd(SysError),
    /// Cloning kill eventfd failed.
    CloneKillEventFd(SysError),
    /// Error while polling for events.
    PollError(SysError),
```
still use `SysError` and are also `EventFd` related errors. Shouldn t those be replaced too?

Reply:
Good eye; done :)